rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1017,6 +1017,49 @@ private void logUpdateCollections(Transaction rskTx) {
         eventLogger.logUpdateCollections(sender);
     }
 
+    protected void processValidationFailure(Federation proposedFederation) {
+        eventLogger.logCommitFederationFailure(rskExecutionBlock, proposedFederation);
+        logger.warn("[processValidationFailure] Proposed federation validation failed so svp values will be cleared.");
+        clearSvpValues();
+    }
+
+    private void clearSvpValues() {
```

Comment:
Should this maybe be part of BridgeStorageProvider class? That would imply adding tests to it 😬

Reply:
mmmjjjjjjj

Reply:
updated